package.json

```diff
@@ -10,6 +10,7 @@
     "git:undo": "git reset --soft HEAD~1",
     "install:dev": "yarn lerna run pip:install:dev --concurrency 1 && pip install -e .",
     "install:prod": "yarn lerna run pip:install:prod --concurrency 1 && pip install .",
+    "install:subpackage": "yarn lerna run pip:install:test -- --scope=$SUBPACKAGE --include-filtered-dependencies --concurrency 1",
```

Comment:
@Centrus007 this command is really cool by the way! :). So glad we decided to hone in on using lerna with yarn. This takes a package defined by scope, and then runs a given command in that package and the packages that package depends upon. And the order it runs them in is dependency order, starting at the bottom of the tree.

So if scope is defined as pymedphys_analysis it will run the command in the following packages on the following order:

* utilities
* coordsandscales
* dicom
* analysis

So, this is the perfect way to just install a subpackages and its dependencies only.